com.cerebralnomad.recipescribe.yml

```diff
@@ -0,0 +1,25 @@
+id: com.cerebralnomad.recipescribe
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: recipe_scribe-2.0.1
+modules:
+  - name: recipescribe
+    buildsystem: simple
+    build-commands:
+      - rm README.txt
+      - install -D recipe_scribe-2.0.1 /app/bin/recipe_scribe-2.0.1
```

Comment:
You install a binary, so how does this work on both x86_64 and aarch64?

Reply:
Would I need something like

build-args:
   -  --arch=x86_64

Reply:
https://docs.flathub.org/docs/for-app-authors/maintenance#only-build-on-x86_64

Reply:
how about building from source instead?

Reply:
better question, if I started over and built from the source files instead of the binary, would the resulting flatpak run on either architecture?

Reply:
I reworked it all to install from the source files instead. Had to pull in Tkinter as well. I used the binary initially to avoid that, I hope I did it correctly. I borrowed some code from another flatpak yaml for that part.
Thanks for your patience, this is my first try and I'm learning as I go.